### PR TITLE
Add/update US student loan sites

### PR DESCRIPTION
**Pull request by Rabadash8820**
See discussion on 2factorauth/twofactorauth#8301

I have only worked with Nelnet and Aidvantage personally, but it might be worth adding/updating the other student loan servicers in this PR. Here's the [official list](https://studentaid.gov/manage-loans/repayment/servicers#your-servicer) (at least I think...can't quite tell if it's exhaustive).

### CODE REVIEW

File: entries/a/aidvantage.studentaid.gov.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "Aidvantage": {
+    "domain": "aidvantage.studentaid.gov",
+    "img": "aidvantage.studentaid.gov.png"
```
Comment (hkamran80) [inline entries/a/aidvantage.studentaid.gov.json]:
```suggestion
    "img": "aidvantage.studentaid.gov.png",
```

File: entries/a/aidvantage.studentaid.gov.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+  "Aidvantage": {
+    "domain": "aidvantage.studentaid.gov",
+    "img": "aidvantage.studentaid.gov.png",
+    "contact": {
+      "form": "https://myaccount.aidvantage.studentaid.gov/EmailUs/SendEmail"
```
Comment (hkamran80) [inline entries/a/aidvantage.studentaid.gov.json]:
```suggestion
      "form": "https://aidvantage.studentaid.gov/contact-us"
```

The page you linked requires authentication.

File: entries/a/aidvantage.studentaid.gov.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    },
+    "categories": [
+      "finance"
+    ],
+    "regions": [
+        "us"
```
Comment (hkamran80) [inline entries/a/aidvantage.studentaid.gov.json]:
```suggestion
      "us"
```

File: entries/n/nelnet.studentaid.gov.json [hunk 4]
```diff
@@ -0,0 +1,12 @@
+{
+  "Nelnet": {
+    "domain": "nelnet.studentaid.gov",
+    "contact": {},
+    "categories": [
+      "finance"
+    ],
+    "regions": [
+        "us"
```
Comment (hkamran80) [inline entries/n/nelnet.studentaid.gov.json]:
```suggestion
      "us"
```

**Comment by hkamran80** [comment]
Please don't force push. It makes it difficult to review PRs.

File: entries/n/nelnet.studentaid.gov.json [hunk 5]
```diff
@@ -0,0 +1,12 @@
+{
+  "Nelnet": {
+    "domain": "nelnet.studentaid.gov",
+    "contact": {},
```
Comment (hkamran80) [inline entries/n/nelnet.studentaid.gov.json]:
```suggestion
    "contact": {
      "form": "https://nelnet.studentaid.gov/content/contactus"
    },
```

They have a live chat option.

**Review by hkamran80 — CHANGES_REQUESTED** [review]

**Comment by Rabadash8820** [comment]
Sorry, I was just trying to knock out some quick fixes for the PR checks before anybody left a comment 😅

File: entries/n/nelnet.studentaid.gov.json (hunk 5 above)
Comment (Rabadash8820) [reply entries/n/nelnet.studentaid.gov.json]: Good find!

File: entries/a/aidvantage.studentaid.gov.json (hunk 1 above)
Comment (Rabadash8820) [reply entries/a/aidvantage.studentaid.gov.json]: Already fixed in my force-pushed commit 😬

**Comment by hkamran80** [comment]
> but it might be worth adding/updating the other student loan servicers in this PR.

Let's add those in another PR.

**Review by hkamran80 — APPROVED** [review]

**Pull request merged**